web/kweb.cfg

```diff
@@ -97,6 +102,44 @@
     }
     folderish /kapi-playground
 
+
+    # Kwmserver
+    proxy /upstreams/kwmserver/ {
+        without /upstreams/kwmserver/
+        upstream kopano_kwmserver:8778
+        policy least_conn
+        health_check /health-check
+        fail_timeout 10s
+        try_duration 30s
+        keepalive 100
+        transparent
+        websocket
+    }
+    ratelimit * 100 200 minute {
+        /upstreams/kwmserver/
+        whitelist 127.0.0.1/8
+    }
+    rewrite /api/v1/websocket {
+        to /upstreams/kwmserver/{path}
+    }
+    rewrite /api/v1/rtm.connect {
+        to /upstreams/kwmserver/{path}
+    }
+    rewrite /api/v1/rtm.turn {
+        to /upstreams/kwmserver/{path}
+    }
```

Comment:
You do not need the kwmserver api/v1/ endpoints for Meet - so best get rid of them. Meet only uses api/kwm/v2

Reply:
cleaned from config